src/passes/RemoveUnusedBrs.cpp

```diff
@@ -88,9 +88,6 @@ static bool canTurnIfIntoBrIf(Expression* ifCondition,
 //  * https://github.com/WebAssembly/binaryen/issues/5983
 const Index TooCostlyToRunUnconditionally = 8;
 
-static_assert(TooCostlyToRunUnconditionally < CostAnalyzer::Unacceptable,
-              "We never run code unconditionally if it has unacceptable cost");
```

Comment:
There are still instructions we know should not be run unconditionally if it can be avoided; is there something else we can replace this assertion with?

Reply:
Good idea, done.